hetrixtools_agent.sh

```diff
@@ -231,10 +231,10 @@ do
 	# Port connections
 	if [ ! -z "$ConnectionPorts" ]
 	then
-		netstat=$(netstat -ntu)
+		netstat=$(netstat -ntu | awk '{print $4}')
 		for cPort in "${ConnectionPortsArray[@]}"
 		do
-			Connections[$cPort]=$(echo | awk "{ print ${Connections[$cPort]} + $(echo "$netstat" | grep -w "$cPort" | wc -l) }")
+			Connections[$cPort]=$(echo | awk "{ print ${Connections[$cPort]} + $(echo "$netstat" | grep ":$cPort" | wc -l) }")
```

Comment:
```suggestion
			Connections[$cPort]=$(echo | awk "{ print ${Connections[$cPort]} + $(echo "$netstat" | grep ":$cPort$" | wc -l) }")
```
Should include `$` to avoid cases like for example: port 33 would match IP:3306 lines.